src/test/java/seedu/address/storage/StorageManagerTest.java

```diff
@@ -68,6 +68,19 @@ public void addressBookReadSave() throws Exception {
     }
 
     @Test
+    public void addressBookBackupReadSave() throws Exception {
+        /*
+         * Note: This is an integration test that verifies the StorageManager is properly wired to the
+         * {@link XmlAddressBookStorage} class.
+         * More extensive testing of UserPref saving/reading is done in {@link XmlAddressBookStorageTest} class.
+         */
+        AddressBook original = getTypicalAddressBook();
+        storageManager.backupAddressBook(original);
+        ReadOnlyAddressBook retrieved = storageManager.readAddressBook(storageManager.getAddressBookFilePath() + "-backup.xml").get();
```

Comment:
Should apply a word-wrap to this line.

Travis check reported: _Line is longer than 120 characters (found 134). [LineLength]_